project/threads/models.py

```diff
@@ -1,3 +1,543 @@
+import os
+import json
+import math
+import datetime
+from calendar import month_name
+
+from accounts.models import Profile
+from common.utils import PathAndRename
+from core.constants import CIVI_TYPES, US_STATES
+from django.conf import settings
+from django.core.files.storage import default_storage
 from django.db import models
+from taggit.managers import TaggableManager
+from django.core.serializers.json import DjangoJSONEncoder
+
+
+class Fact(models.Model):
+    body = models.CharField(max_length=511)
+
+    created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
+
+
+class Category(models.Model):
```

Comment:
In issue #1032,  @brylie mentioned that 

> category - although this might be better in a categories app since it is shared between threads and accounts apps

You may not add `Category` model in `threads/models.py`.